src/include/ipc/info.h

```diff
@@ -110,8 +110,8 @@ struct sof_ipc_window {
 	struct sof_ipc_window_elem window[];
 } __attribute__((packed));
 
-/* extended data, compiler version */
-struct sof_ipc_cc_version {
+/* sof_ipc_cc_version struct without flex array at the end */
```

Comment:
dont see where this flex array is removed ?

Reply:
`struct sof_ipc_cc_version` is still here, just below `sof_ipc_cc_version_base` and from host side `sof_ipc_cc_version` doesn't change